<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sale_form</class>
 <widget class="QFrame" name="sale_form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1246</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="bill_number_date">
     <property name="text">
      <string>Накладная №1 от 10.10.10</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Получатель</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="reciever_combo">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Пациент</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Медицинская организация</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0" colspan="6">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="save_close_btn">
     <property name="maximumSize">
      <size>
       <width>190</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Провести накладную</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="save_btn">
     <property name="maximumSize">
      <size>
       <width>190</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Сохранить и не проводить</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="minimumSize">
      <size>
       <width>190</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>96</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Просто кнопка</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QPushButton" name="close_btn">
     <property name="maximumSize">
      <size>
       <width>190</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Закрыть без сохранения</string>
     </property>
    </widget>
   </item>
   <item row="6" column="5">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Всего лс на сумму 0.00р</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="reciever_btn">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Рецепт</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="reciever_label">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QComboBox" name="financing_combo"/>
   </item>
   <item row="4" column="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Источник финансирования</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>close_btn</sender>
   <signal>clicked()</signal>
   <receiver>sale_form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>724</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>529</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
